core-concepts/wallets/veworld/user-guide/wallet.md

```diff
@@ -0,0 +1,244 @@
+
+**Steps for a new wallet:**
+
+1. Tap the **Create new wallet** button
+2. Read and store your mnemonic
+3. Mark the checkbox that states you saved your mnemonic somewhere
+4. Pass the quiz with the 3 words from your mnemonic
+{% endtab %}
+
+{% tab title="Extension" %}
+| Fresh Start                                                                                                                                                                                                                                                      | Add a new one                                                                                                                                                                                                                                                                                                                                                |
+| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| <p>The first time you install the extension you should:</p><ol><li>Follow and complete or skip the <em>welcome wizard</em></li><li>Type your favorite password twice and then click the <strong>CONFIRM</strong> button</li><li>Follow the below steps</li></ol> | <p>If you already have a wallet on VeWorld, and you want to create a new one:</p><ol><li>From your dashboard tap the top-right 3-dots icon <img src="../../../../.gitbook/assets/veworld/more_vert.svg" alt=""></li><li>Select <strong>Manage Wallets</strong></li><li>Click the <strong>Add Wallet</strong> button</li><li>Follow the below steps</li></ol> |
+
+**Steps for a new wallet:**
+
+1. Click the **Create wallet** button
+2. Follow and complete or skip the _security wizard_
+3. Read and store your mnemonic
+4. Mark the checkbox that states you saved your mnemonic somewhere
+5. Pass the quiz with the 3 words from your mnemonic
+6. Click the **GO TO HOMEPAGE** button
+{% endtab %}
+{% endtabs %}
+
+## Import a wallet
+
+### Import a local wallet
+
```

Comment:
Were the "mnemonic" and "private key" concepts introduced before to the average user?
While for "keystore" I imagine there is/will be a doc explaining how to create a backup of such kind, right?

Reply:
These concepts had not been introduced before. Keystore export is not available on VeWorld, but developers can create it from commanline I guess. So I think it's a feature for advanced users. @victhorbi